tests/README.md

```diff
@@ -111,24 +110,29 @@ go test -v -timeout 1800s -run SmokeSuite github.com/rook/rook/tests/integration
 
 ##### To run specific without installing rook
 ```
-go test -v -timeout 1800s -run SmokeSuite github.com/rook/rook/tests/integration --skip_install_rook=true
+go test -v -timeout 1800s -run SmokeSuite github.com/rook/rook/tests/integration --skip_install_rook
 ```
 If the `skip_install_rook` flag is set to true, then rook is not uninstalled either. 
 
 #### Run Longhaul Tests
-Using the `go test -count n` option, any tests can be repeated `n` number of times to simulate load or longhaul test. Although 
-any test can be converted to a longhaul test, it's ideal to run integration tests under load for an extended period. Also a new custom test flag `load_parallel_runs` is added to control the number of concurrent operations being performed.
-For example look at the [block long haul test](/tests/block/k8s/longhaul/basicBlockonghaul_test.go)
- 
- To run a longhaul test you can run any integration test with `-count` and `--load_parallel_runs` options
- e.g.
+[longhaul](/tests/block/k8s/longhaul) Tests are integration tests that run for extended period of time. A load profile can be configured
+using the following load test flags 
++ `load_parallel_runs`: controls number of concurrent operations reading and writing data
++ `load_time` : controls how the long the test runs for
++ `load_size` : controls size of data and 
++ `load_volumes` : controls the number of concurrent volumes under test(file,block or object)
+  e.g.
  ```
- go test -run TestK8sBlockLongHaul github.com/rook/rook/tests/longhaul --load_parallel_runs=20 -count=1000
+ go test -run TestObjectLongHaul github.com/rook/rook/tests/longhaul --load_parallel_runs=20 --load_time 1800 --load_size S --load_volumes 3
  ```
  The Longhaul test just like other test is going to install rook if it's not already installed, but it is not going to clean up test data or uninstall rook after the run. 
  Longhaul test is designed to run multiple times on the same setup and installation of rook to tests its stability. Test Data and rook should be cleaned up manually after the test.
  
  
+ You can measure memory,cpu,ipos,throughput etc. on a cluster, node and pod level using Prometheus and visualize it using Grafana. This would be useful in measuring performance and health of the cluster.
```

Comment:
`You can measure memory, cpu, IOPS, throughput, and other node and pod metrics using Prometheus. The metrics collected during the tests can be visualized with Grafana.`

Reply:
Done.